source/B2BApi/Migration/ITimeSeriesJsonToEbixStreamWriter.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.EDI.OutgoingMessages.Domain.Models.MarketDocuments;
+
+namespace Energinet.DataHub.EDI.B2BApi.Migration;
+
+public interface ITimeSeriesJsonToEbixStreamWriter
```

Comment:
There seems to be a lot of "time series" or "metered data" mentioned in these new files. Would it make sense to rename them to `Measurements`?

```suggestion
public interface IMeasurementsJsonToEbixStreamWriter
```